thirdparty/thrust/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+# This file is part of the Acts project.
```

Comment:
Do we expect that we'll want to use Thrust directly in Acts? :thinking: If not, we could leave it up to traccc to build whichever version it wants. (Still telling Detray that it wouldn't build it, so that only traccc does.)

Reply:
I don't think we'll use Thrust directly in Acts. Should I change it so that traccc sets it up for itself and detray?

Reply:
I think thrust will currently always be coupled to the traccc + detray, let's leave it with them for the moment.